modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/AssignQualityScore.pm

```diff
@@ -98,6 +104,30 @@ sub write_output {
     close $out_fh;
 
     $self->warning("Scores written to $output_file!");
+
+    foreach my $mlss_id ( @aln_mlss_ids ) {
+        my $mlss = $mlss_adaptor->fetch_by_dbID($mlss_id);
+        print "Writing n_${member_type}_wga_score to the database\n" if $self->debug;
+        $self->_write_n_tag($mlss, "${member_type}_wga", \%max_quality);
+        print "Tag: n_${member_type}_wga_score written!\n\n" if $self->debug;
+    }
+}
+
+sub _write_n_tag {
+    my ($self, $mlss, $label, $scores) = @_;
+
+    my %distrib_hash;
+    foreach my $score ( values %$scores ) {
+        my $floor_score = int($score/25)*25;
```

Comment:
```suggestion
        my $floor_score = $score - ($score % 25);
```
This is a suggestion only: this alternative is much faster (20 times) in Python (and I assume the same for Perl, as there are less and simpler operations involved).

Reply:
As I was simply moving this over from the highConfidence pipeline I didn't see the need to alter it, but if everyone else agrees, I can make the change?

Reply:
I doesn't work, you still need to add an `int` somewhere to truncate the value. Otherwise it returns 0.5 for 1.5 (instead of 0), 0.9 for 24.9 (instead of 0), 25.1 for 25.1 (instead of 25), etc.
And also, more importantly, the performance should be assessed in Perl, not Python.